harness/determined/cli/resource_pool.py

```diff
@@ -0,0 +1,166 @@
+from argparse import ONE_OR_MORE, Namespace
+from typing import Any, List
+
+from determined.cli import render, setup_session
+from determined.common.api import authentication, bindings
+from determined.common.declarative_argparse import Arg, Cmd
+
+
+def get_workspaces_string(workspaces: List[str]) -> str:
+    workspaces_str = ""
+    for workspace in workspaces[:-1]:
+        workspaces_str += workspace + ", "
+
+    if workspaces_str != "":
+        workspaces_str += "and "
+    workspaces_str += workspaces[-1]
+    return workspaces_str
+
+
+@authentication.required
+def add_binding(args: Namespace) -> None:
+    body = bindings.v1BindRPToWorkspaceRequest(
+        resourcePoolName=args.pool_name, workspaceNames=args.workspace_names
+    )
+    bindings.post_BindRPToWorkspace(setup_session(args), body=body, resourcePoolName=args.pool_name)
+
+    workspaces_string = get_workspaces_string(args.workspace_names)
+
+    print(
+        f'added bindings between the resource pool "{args.pool_name}" '
+        f"and the following workspaces: {workspaces_string}"
```

Comment:
Please ignore me if this is specified in the PRD, but as a user with the set of personal biases and expectations that I have I'd prefer reading this output formatted simply:
```python
f"and the following workspaces: {args.workspace_names}"
```
which I'm expecting would render as
```
and the following workspaces: ['workspace1', 'workspace2', 'workspace3']
```

Plus (bonus), less code.

Just an opinion.

Reply:
sounds good, that looks like a no brainer to me then!